src/containers/JellyfishServer.tsx

```diff
@@ -0,0 +1,33 @@
+import App from "./App";
+import { ServerSDKProvider } from "../components/ServerSdkContext";
+import { RoomsContextProvider } from "./RoomsContext";
+import { ApiProvider } from "./Api";
+
+export type ServerProps = {
+  host: string;
+  protocol: string;
+  path: string;
+  serverToken: string;
+  refetchDemand: boolean;
+  active: boolean;
+};
+
+export const JellyfishServer = ({ host, protocol, path, serverToken, refetchDemand, active }: ServerProps) => {
+  return (
+    <ServerSDKProvider
+      currentHost={host}
+      currentPath={path}
+      currentProtocol={protocol}
+      currentServerToken={serverToken}
+    >
+      <RoomsContextProvider>
+        <ApiProvider>
+          <div className="flex flex-col">
+            <App host={host} refetchDemand={refetchDemand} active={active} />{" "}
+            {/* TODO: remove active, not needed with other PR*/}
```

Comment:
Please remove `{" "}` and I don't understand this TODO. Please remove it or explain it.

Reply:
I've essentially done what was said to be wrong in the rooms case. I'm hiding servers that are currently unused to prevent dismounting. Due to small number of servers I decided it is not that bad for now, I'm willing to change it if necessary though.